### ISSUE TITLE
Hard-coded Database Name "SQLWATCH" in DACPAC file

### ISSUE THREAD

**Issue by koglerk** [marcingminski/sqlwatch#60]
**Describe the bug**
Hard-coded database name still exists in the postdeploy script. 

**To Reproduce**
1. Download [the dacpac](https://github.com/marcingminski/sqlwatch/blob/development/SQLWATCHDB/bin/Debug/SQLWATCH.dacpac) from the development branch 
2. Double-click on the dacpac to extract the files
3. Open the postdeploy.sql file extracted from the dacpac
4. Line 844: `@database_name=N'SQLWATCH', `

**Expected behavior**
Line 844 should be `@database_name=@database,`

**Comment by marcingminski** [comment]
Thank you for raising this and apologise for the delay in answering. 

I feel this may be an occurring problem since Dacs are binary files and we all know it's not the best idea to keep binary files in source control. 

The only reason I keep binaries in branches is so they are easily accessible (mainly for dbatools installation :) ) as the path does not change. 

I have however considered two options:

 ### AppVeyor Hosting
I use AppVeyor for build and deployment testing and this does generate Dac. They also host artefacts for 3 months (may not be long enough)

```
https://ci.appveyor.com/api/projects/<account>/<project>/artifacts/<artifact_file_path>
```

### GitHub releases
AppVeyor can automatically create GitHub releases (I got this setup but not used) and can push generated Dac as an attachment. With GitHub releases, the path will contain release version so will not be static. However, this could actually allow downloading/installing the previous version on-demand. 

To can access latest GitHub releases programmatically via PowerShell:

```
https://api.github.com/repos/marcingminski/sqlwatch/releases/latest
```

There also a good, working example:
https://gist.github.com/MarkTiedemann/c0adc1701f3f5c215fc2c2d5b1d5efd3

Thoughts?

**Comment by marcingminski** [comment]
Having said that, I will fix the dev branch and will keep Dac in branches until we have a better option.

**Comment by koglerk** [comment]
Oh, I finally understand why the dacpac wasn't updating! 

It seems pretty easy to use the second solution above to automatically download the latest GitHub release. This would prevent us from being able to select the branch to download from, but that's a small piece to give up. I'll get this done in the next few days.

**Comment by marcingminski** [comment]
addressed in 9a3d8b60a47c47415494a932c74349dad1ef3e76. DACPACs will now be only available in releases.
